tools/depth-quality/depth-quality-model.cpp

```diff
@@ -654,37 +654,34 @@ namespace rs2
 
                         try
                         {
-                            if (_depth_sensor_model)
+                            auto && ds = _depth_sensor_model->dev.first< depth_sensor >();
```

Comment:
The tool won't yell about `_depth_sensor_model->`?

Reply:
It is already checked (at line 536), the tool showed this test as redundant